hackathon_site/dashboard/frontend/src/components/general/PopupModal/PopupModal.test.tsx

```diff
@@ -0,0 +1,85 @@
+import { render } from "../../../testing/utils";
```

Comment:
relative import here, change to absolute

Reply:
done